tests/model_registry/scc/utils.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+from typing import Any
+
+from ocp_resources.resource import NamespacedResource
+
+
+def get_uid_from_namespace(namespace_scc: dict[str, str]) -> str:
+    return namespace_scc["uid-range"].split("/")[0]
+
+
+def validate_pod_security_context(
+    pod_security_context: dict[str, Any],
+    namespace_scc: dict[str, str],
+    model_registry_resource: NamespacedResource,
+    ns_uid: str,
+) -> list[str]:
+    errors = []
+    pod_selinux_option = pod_security_context.get("seLinuxOptions", {}).get("level")
+    if pod_selinux_option != namespace_scc["seLinuxOptions"]:
+        errors.append(
+            f"selinux option from pod {model_registry_resource.name} {pod_selinux_option},"
+            f" namespace: {namespace_scc['seLinuxOptions']}"
+        )
+    if pod_security_context.get("fsGroup") != int(ns_uid):
```

Comment:
Same as the other comment, uid is defined as a range, I would check that whatever id is assigned here falls within the range rather than expecting a specific one